drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -565,16 +569,26 @@ static int bt_ipm_open(const struct device *dev, bt_hci_recv_t recv)
 			K_PRIO_COOP(CONFIG_BT_DRIVER_RX_HIGH_PRIO),
 			0, K_NO_WAIT);
 
+	hci->recv = recv;
+
+	LOG_DBG("IPM Channel Open Completed");
+
+	return 0;
+}
+
+static int bt_ipm_setup(const struct device *dev, const struct bt_hci_setup_params *params)
+{
+	ARG_UNUSED(params);
+	int err;
```

Comment:
```suggestion
	int err;

	ARG_UNUSED(params);
	ARG_UNUSED(dev);
```

Reply:
Yes, Let's do it for all the unused params